<launch>
    <!-- Set the initial pose of the robot -->
    <param name="InitialX" type="double" value="0" />
    <param name="InitialY" type="double" value="0" />
    <param name="InitialTheta" type="double" value="0" />

    <param name="Largh" type="double" value="0.169" />
    <param name="Lungh" type="double" value="0.2" />
    <param name="WheelRad" type="double" value="0.07" />

    <node pkg="RoboticsProject" type="EncoderParser" name="EncoderParser"/>
    <node pkg="RoboticsProject" type="KinematicCore" name="KinematicCore"/>
    <node pkg="RoboticsProject" type="OdometryCore" name="OdometryCore"/>

</launch>